mergeAOSP.sh

```diff
@@ -15,23 +15,33 @@ COMMIT=$1
 
 git merge $COMMIT --no-commit --no-ff || true
 
```

Comment:
I tried to move files into `scripts` folder, but it seems not very convenient to use

Reply:
The scripts folder was exactly my comment. Root is not a good place for all these scripts. Also, a script shouldn't depend on the current user directory - now it works correctly ONLY if manually/explicitly "`cd`" to root first, but THIS is not convenient and based on assumptions.


The generic pattern is pretty simple:
```sh
ROOT_DIR="$(dirname "$0")/.."
(cd $ROOT_DIR; git merge ...) // Change dir for only one command
```

Or just use `--git-dir` parameter

Reply:
moved to "scripts"